src/main/java/name/admitriev/jhelper/configuration/TaskConfigurationType.java

```diff
@@ -19,7 +19,7 @@ public TaskConfigurationType() {
 					@NotNull
 					@Override
 					protected Icon compute() {
-						return IconLoader.getIcon("/name/admitriev/jhelper/icons/task.png");
```

Comment:
This is because the `resources/icons/*.png` are directly accessible as `icons/icon.png`

Reply:
I meant why you put it at resources/icons/*.png and not in resources/name/admitriev/jhelper/icons/*.png

Reply:
That is the recommended style written in the Jetbrains plugin page.

Reply:
I see, they indeed use it in their example.
In my opinion, using whole path makes more sense because it clearly avoids any name clashes (but I am open to discussion potentially)

What I'm sure is that it's irrelevant to the switch from devkit to gradle.

Reply:
I have made changes as u said now.